zerver/models/messages.py

```diff
@@ -621,7 +621,7 @@ def get_usermessage_by_message_id(
     user_profile: UserProfile, message_id: int
 ) -> Optional[UserMessage]:
     try:
-        return UserMessage.objects.select_related().get(
+        return UserMessage.objects.select_related("user_profile", "message").get(
```

Comment:
Can you put this in its own commit with its own commit message? Could be appropriate for a prep PR; @sahil839 FYI.

Reply:
Created new commit for this change.

Reply:
I think we can remove the `select_related` call itself, as we probably do not access this field from the `UserMessage` object returned by this function. Would be good if you could verify this as well.

Reply:
Removed ```select_related``` call.